htmresearch/algorithms/simple_union_pooler.py

```diff
@@ -64,21 +68,27 @@ def updateHistory(self, activeCells):
       self._activeCellsHistory.pop(0)
```

Comment:
Don't think it is worth the trouble.  The list typically has about 10 items in it.  We can optimize if this operation shows up on the profile.  Currently the entire region is 0.4% of the total runtime.